### DISCUSSION TITLE
Integrating an Application-Based Review Process within Pretix

### DISCUSSION THREAD

**Discussion by theschweitz** [pretix/pretix#2956] [Ideas]
We think it would be a great feature for the Pretix team to integrate an application-based review process within Pretix, similar to how this all takes place within Pretalx. 

Currently there is a feature "Order approvals" (photo attached) where organizers can put a product up for sale that requires the event organizer to Approve the order before the payment is accepted & the order can be allowed to proceed. 
According to the Pretix docs, the team also envisions this feature around the same use case ("If you do not know your audience already, but still want to restrict it to a certain group, e.g. people with a given profession...").

We envision a process where users can apply for a Discounted Ticket by filling out all required questions, then we could review submissions as we do on Google Sheets — narrowing down applicants, scoring/categorizing them by most qualified, and still being able to grant tickets to applicants of lesser quality/lower rating. Then we can approve/deny the orders.

![order approvals](https://user-images.githubusercontent.com/77123385/206280022-5646e8d2-ae47-46e2-a99f-e3f62d192b60.png)

**Comment by raphaelm** [comment]
Yes, the approval process can be used like this but we have no plans to add a full review/scoring system into pretix. We know from our pretalx friends how complex that can get and since it's not close to pretix' core functionality I don't think the complexity os warranted here. Maybe someone else wants to create a plugin, there should be not much issue adding this as an additional view with some additional models
